src/Cohesity.Powershell/Scripts/Utility/Invoke-RestApi.ps1

```diff
@@ -101,13 +101,15 @@ function Invoke-RestApi
         if(401 -eq $Global:CohesityAPIError.StatusCode.Value__) {
             if($true -eq $Global:CohesityCmdletConfig.RefreshToken) {
                 Write-Host "The session token has expired, attempting to refresh."
+				$credentialsJson = [Environment]::GetEnvironmentVariable('cohesityCredentials', 'Process')
                 $cohesitySession = Get-Content -Path $HOME/.cohesity | ConvertFrom-Json
-                if($null -ne $cohesitySession.Credentials) {
+                if($null -ne $credentialsJson) {
                     $cohesityUrl = $cohesitySession.ClusterUri + "/irisservices/api/v1/public/accessTokens"
+					$credentialsObject = $credentialsJson | ConvertFrom-Json
                     $payload = @{
-                        Domain                 = $cohesitySession.Credentials.Domain
-                        Username                   = $cohesitySession.Credentials.Username
-                        Password                   = $cohesitySession.Credentials.Password
+                        Domain                 = $credentialsObject.Domain
```

Comment:
Good stuff, what about the C# based cmdlets ?

Reply:
The C# cmdlete has been covered on the RestApiClient.cs

Reply:
LGTM